app/src/Application/DbSettingsObj.h

```diff
@@ -4,46 +4,45 @@
 
 #include <QtCore/QtCore>
 
-namespace Status::Application {
+namespace Status::Application
+{
 
-    class DbSettingsObj: public QObject
```

Comment:
Main namespace indentation? it feels so wasted and repetitive spacing 😔

Reply:
It removed the indentation :sweat_smile: